browser/resources/settings/email_aliases_page/tests/email_aliases_modal.test.tsx

```diff
@@ -301,4 +306,109 @@ describe('EmailAliasModal', () => {
       expect(mockOnReturnToMain).toHaveBeenCalled()
     })
   })
+
+  it('shows error message when creating alias fails', async () => {
+    mockEmailAliasesService.updateAlias = jest.fn().mockImplementation(
+      () => Promise.resolve({ errorMessage: 'emailAliasesUpdateAliasError' }))
+
+    const mockEditState: EditState = {
+      mode: 'Create',
+      alias: {
+        email: 'new@brave.com',
+        note: 'New Alias',
+        domains: undefined
+      }
+    }
+
+    render(
+      <EmailAliasModal
+        editState={mockEditState}
+        mainEmail={mockEmail}
+        aliasCount={0}
+        onReturnToMain={mockOnReturnToMain}
+        emailAliasesService={mockEmailAliasesService}
+      />
+    )
+
+    // Wait for error message to be displayed. Create button should be disabled.
+    const saveButton = screen.getByText('emailAliasesCreateAliasButton')
+    await waitFor(() => {
+      expect(saveButton).toHaveAttribute('isdisabled', 'false')
+    })
+
+    await act(async () => {
+      // Click save button
+      saveButton.shadowRoot?.querySelector('button')?.click()
+    })
+
+    // Check that updateAlias was called and returned an error
+    await waitFor(() => {
+      expect(mockEmailAliasesService.updateAlias).toHaveBeenCalled()
+      expect(mockEmailAliasesService.updateAlias).toHaveReturnedWith(
+        Promise.resolve({
+          errorMessage: 'emailAliasesUpdateAliasError'
+        })
+      )
```

Comment:
nit: remove this - no point checking it returns what we said it'll return :smile:

Reply:
fixed